Penumbra/Import/Textures/CombinedTexture.Manipulation.cs

```diff
@@ -206,26 +243,59 @@ private void MultiplyPixelsRight(int y, ParallelLoopState _)
         }
     }
 
+    private byte[] ResizePixels(byte[] rgbaPixels, int sourceWidth, int sourceHeight)
+    {
+        if (sourceWidth == _targetWidth && sourceHeight == _targetHeight)
+            return rgbaPixels;
+
+        byte[] resizedPixels;
+        using (var image = Image.LoadPixelData<Rgba32>(rgbaPixels, sourceWidth, sourceHeight))
+        {
+            image.Mutate(ctx => ctx.Resize(_targetWidth, _targetHeight));
```

Comment:
Decided to prefer default (bicubic) to Lanczos3 by choice or missed it?

Reply:
Missed it, I've gotta study ImageSharp's docs more.